client/session_bootstrap_blocks_test.go

```diff
@@ -1159,7 +1161,7 @@ func (m *fetchBlocksReqMatcher) Matches(x interface{}) bool {
 	}
 
 	for i := range params {
-		if !params[i].id.Equal(ts.BinaryID(req.Elements[i].ID)) {
+		if params[i].id.String() != string(req.Elements[i].ID) {
```

Comment:
Yeah, mainly just because this in tests and I didn't want to have to copy the bytes behind `req.Elements[i].ID` to be safe so thought it would be simpler to just stringify them.  I can copy the bytes however.